iOS/Flows/Home/Members/MembersCollectionView.swift

```diff
@@ -33,20 +34,8 @@ class MembersCollectionView: CollectionView {
     
     func reveal() {
         Task.onMainActorAsync {
-            
-            var offset: CGFloat = 1.2
-
-            switch ScreenSize.current {
-            case .phoneSmall:
-                offset = 0.8
-            case .phoneMedium:
-                offset = 1.1
-            default:
-                offset = 1.2
-            }
-
             let centerOffsetX = (self.contentSize.width - self.frame.size.width) / 2
-            let centerOffsetY = ((self.contentSize.height - self.frame.size.height) * offset) / 2
```

Comment:
Do we really need this? It makes the centering logic more complicated and I'm not sure what it gets us. We can put it back if there's a reason.

Reply:
No that's legacy.